pixi.toml

```
@@ -39,10 +40,13 @@ pre-commit = ">=3.7.1,<4"
 pre-commit-hooks = ">=4.6.0,<5"
 ruff = ">=0.4.8,<0.5"
 typos = ">=1.23.1,<2"
+mypy = ">=1.10.1,<1.11"
```

```suggestion
mypy = ">=1.10.1,<2"
```

We are fine with non-breaking releases, I think